src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -335,10 +338,14 @@ function ReportActionItemMessageEdit(props) {
      * @param {String} emoji
      */
     const addEmojiToTextBox = (emoji) => {
-        setSelection((prevSelection) => ({
-            start: prevSelection.start + emoji.length + CONST.SPACE_LENGTH,
-            end: prevSelection.start + emoji.length + CONST.SPACE_LENGTH,
-        }));
+        let newSelection = {
```

Comment:
```suggestion
        const newSelection = {
```

Reply:
Thanks, addressed in 7fc489c77a22ea300367d5c295ee396de5c24668